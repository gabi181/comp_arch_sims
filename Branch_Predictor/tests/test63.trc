16 5 4 1 local_history global_tables not_using_share
0x45407AB4 N 0x68A30564
0x6C5CCCAC T 0x0A945924
0x6C5CCCAC T 0x0A945924
0x45407AB4 T 0x68A30564
0x45407AB4 N 0x68A30564
0x45407AB4 T 0x68A30564
0x6C5CCCAC T 0x0A945924
0x45407AB4 T 0x68A30564
0x6C5CCCAC N 0x0A945924
0x45407AB4 T 0x68A30564
0x6C5CCCAC N 0x0A945924
0x45407AB4 T 0x68A30564
0x6C5CCCAC T 0x0A945924
0x45407AB4 T 0x68A30564
0x45407AB4 T 0x68A30564
0x45407AB4 N 0x68A30564
0x45407AB4 T 0x68A30564
0x45407AB4 T 0x68A30564
0x6C5CCCAC T 0x0A945924
0x45407AB4 N 0x68A30564
0x45407AB4 N 0x68A30564
0x45407AB4 N 0x68A30564
0x45407AB4 N 0x68A30564
0x45407AB4 T 0x68A30564
