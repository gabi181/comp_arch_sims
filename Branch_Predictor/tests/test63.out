0x45407ab4 N 0x45407ab8
0x6c5cccac N 0x6c5cccb0
0x6c5cccac N 0x6c5cccb0
0x45407ab4 N 0x45407ab8
0x45407ab4 T 0x68a30564
0x45407ab4 N 0x45407ab8
0x6c5cccac N 0x6c5cccb0
0x45407ab4 N 0x45407ab8
0x6c5cccac N 0x6c5cccb0
0x45407ab4 N 0x45407ab8
0x6c5cccac N 0x6c5cccb0
0x45407ab4 N 0x45407ab8
0x6c5cccac N 0x6c5cccb0
0x45407ab4 N 0x45407ab8
0x45407ab4 N 0x45407ab8
0x45407ab4 T 0x68a30564
0x45407ab4 N 0x45407ab8
0x45407ab4 N 0x45407ab8
0x6c5cccac N 0x6c5cccb0
0x45407ab4 N 0x45407ab8
0x45407ab4 N 0x45407ab8
0x45407ab4 N 0x45407ab8
0x45407ab4 N 0x45407ab8
0x45407ab4 N 0x45407ab8
flush_num: 17, br_num: 24, size: 704b
